Automattic/wp-calypso
# Simple Payments: add a Guided Tour for end of the year

This PR adds a new tour, telling people about the Simple Payment Button. It should start for people on the Business or Premium plan when they're visiting a page on which the sidebar's "Site Pages" menu item (and it's associated "Add" button) is visible. 

Note: the centered step that comes up when we enter the Editor is there for technical reasons. Unfortunately the Editor takes too long to load / Guided Tours is too impatient to find the target element to attach the step to. 

To launch the tour:

http://calypso.localhost:3000/stats/day/SITE_URL?tour=simplePaymentsEndOfYearGuide

Screen capture:

![happy-holidays](https://user-images.githubusercontent.com/23619/32406739-6e9201a2-c17e-11e7-8630-5d8f74a4710c.gif)

To test:
- launch the tour
- check the copy and flow of the tour
- try to break the tour
- make sure the code looks alright
